lib/storage-client.js

```diff
@@ -243,6 +243,8 @@ class ChromedriverStorageClient {
       await fs.mv(chromedriverPath, dst, {
         mkdirp: true
       });
+      fs.chmod(dst, 0o755);
```

Comment:
await is missing

also, the call should be moved outside of the try/finally block

Reply:
Thank you, It's ok now.